recipes/toad/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "toad" %}
+{% set version = "0.0.58" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9dce218ac11eb9e3a80b233557694d0d349418a1ba517306919dff02595e5f07
+
+build:
+  number: 0
+  skip: True  # [py==27]
+  entry_points:
+    - toad = toad.cli:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy >=1.15.4
```

I gues this is linking against numpy? If not this probably isn't needed.

Assuming it is needed, it is probably enough to use the global pin
```suggestion
    - numpy
```
if not use 
```suggestion
    - numpy 1.15
```

---

Yeah, it is linking against numpy, so it needs to stay. When I tried the global pin, it used 1.14, so that is why I left 1.15.

---

Is it should be this?
```suggestion
    - numpy 1.15
    - {{ pin_compatible('numpy') }}
```

---

Yes, please try that.

---

@npavlovikj @chrisburr I have updated the recipe, please review it again.